package.json

```
@@ -10,7 +10,11 @@
   },
   "devDependencies": {
     "@automattic/color-studio": "^2.5.0",
-    "@babel/eslint-parser": "7.15.4",
+		"@babel/core": "7.28.3",
+		"@babel/eslint-parser": "^7.23.3",
+		"@babel/plugin-transform-nullish-coalescing-operator": "^7.26.6",
+		"@babel/plugin-transform-optional-chaining": "^7.25.9",
+		"@babel/runtime-corejs3": "^7.27.0",
```

Nit: the whitespace looks incorrect here.
```suggestion
    "@babel/core": "7.28.3",
    "@babel/eslint-parser": "^7.23.3",
    "@babel/plugin-transform-nullish-coalescing-operator": "^7.26.6",
    "@babel/plugin-transform-optional-chaining": "^7.25.9",
    "@babel/runtime-corejs3": "^7.27.0",
```

---

Nit: the tabs look like they are back again.
```suggestion
    "@babel/core": "7.28.3",
    "@babel/eslint-parser": "^7.23.3",
    "@babel/plugin-transform-nullish-coalescing-operator": "^7.26.6",
    "@babel/plugin-transform-optional-chaining": "^7.25.9",
    "@babel/runtime-corejs3": "^7.27.0",
```